lib/gui/css/main.css

```diff
@@ -6402,30 +6402,30 @@ body {
 .component-drive-selector-body .list-group-item-text {
   word-break: break-all; }
 
-.modal-drive-selector-modal .list-group {
+.component-drive-selector-body .list-group {
```

Comment:
Maybe someone forgot to `npm run sass` in a previous commit? Running it again changes nothing, and it's based on `master` as far as I'm aware...